scripts/bootstrap-gitea.sh

```diff
@@ -384,17 +384,18 @@ print_urls_passwords () {
     echo "oc get route -n openshift-gitops openshift-gitops-cntk-server -o template --template='https://{{.spec.host}}'"
     echo "oc extract secrets/openshift-gitops-cntk-cluster --keys=admin.password -n openshift-gitops --to=-"
     echo "# -----"
-    echo "# The Cloud Pak console and admin password"
-    echo "oc get route -n ${CP_DEFAULT_TARGET_NAMESPACE} integration-navigator-pn -o template --template='https://{{.spec.host}}'"
-    echo "oc extract -n ibm-common-services secrets/platform-auth-idp-credentials --keys=admin_username,admin_password --to=-"
-    echo "# -----"
     echo "# "
     echo "# Gitea UI: $(oc get route ${INSTANCE_NAME} -n ${TOOLKIT_NAMESPACE}${INSTANCE_NAME} -n ${TOOLKIT_NAMESPACE} -o template --template='https://{{.spec.host}}')"
     echo "# "
     echo "# To get the Gitea admin ID and admin password:"
     echo "# -----"
     echo "oc extract secrets/${INSTANCE_NAME}-access --keys=username,password -n ${TOOLKIT_NAMESPACE} --to=-"
     echo "# -----"
+    echo "# "
+    echo "# The Cloud Pak console and admin password"
+    echo "oc get route -n ${CP_DEFAULT_TARGET_NAMESPACE} integration-navigator-pn -o template --template='https://{{.spec.host}}'"
+    echo "oc extract -n ibm-common-services secrets/platform-auth-idp-credentials --keys=admin_username,admin_password --to=-"
```

Comment:
@lsteck, this is the Platform Navigator route which is specific to CP4I and diff from the Cloud Pak console.  This is only required if CP4I modules in the script are set by an env var and that's currently still a wip.  So we should comment this out for now as CP will not be installed when this script is run.

Reply:
@hollisc I removed.  Please review.  Thanks!